source/source_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package source
+
+import (
+	"context"
+	"strings"
+	"testing"
+)
+
+func TestConfigureSource_FailsWhenConfigEmpty(t *testing.T) {
```

Comment:
I believe we should have more test coverage for a source.

Reply:
You mean this file or another?

Reply:
source_integration_tests.go